src/components/shared/Validations.tsx

```diff
@@ -63,11 +63,11 @@ const createValidationsFormatter = (name: string, options?: { exact?: boolean; n
 };
 
 const validationFormatters: Record<string, (value: unknown) => ValidationFormat | null> = {
-  enum: createValidationsFormatter('Allowed'),
-  examples: createValidationsFormatter('Example'),
+  enum: createValidationsFormatter('Allowed', { nowrap: true }),
```

Comment:
Hmm, and how will one recognized
```
enum: [1, "1"]
```
I don't think it's a good decision to remove quotes. These are strings, so let them be.

Reply:
We can't in such a case. That's why I added it in the first place.
@mnaumanali94 is it really needed to remove those quotes? We're losing a type then.

Reply:
> Hmm, and how will one recognize such an enum?
> 
> ```
> enum: [1, "1"]
> ```

This is a terrible enum! If someone is mixing stringified numbers and numberified numbers in an enum, their API is beyond our help. They're a disgrace to the industry and we should encourage them to use SwaggerHub to hurt SwaggerHub's reputation. :trollface:

Reply:
I know it is, but it's just to show the example.
The below cases are valid tho:

```
enum: ["1", "2", "3"]
```

```
enum: [1, 2, 3]
```

If we render them the same way, how will one be able to tell the difference?

Reply:
@P0lip are enums the only potential problem? Or can you think about more examples where this change would cause confusion? 🤔 

What if we made the enums an exception here? @mnaumanali94 is a strong proponent of that change. 🤷

Reply:
Well, mostly enums since other keywords are not used for validation. 
I'd still keep it as is, though.
IMHO in this particular case we shouldn't prioritize a negligible benefit of improved readability over the potential of causing confusion and loss of valuable information.

I'll leave it to you all, tho.

Reply:
One idea I have in my mind is to make use of colors we use for types. Each example could have a different color based on its type.
That would satisfy all parties

Reply:
The type of enums i.e. String vs Integer should show that information @P0lip or am i missing something?

Reply:
It depends on the scenario.
Consider the below:

```json
{ 
  "enum": ["0", 1, "2"]
}
```

or

```json
{
  "type": ["string", "number"],
  "enum": ["0", 1, "2"]
}
```

One won't be able to tell the difference once we apply the change.

Reply:
Hmm. Have you ever seen an enum like this though? 🤦🏼

Reply:
Should 
```
type: string,
enum: [asc, desc]
```

In our docs, these values should appear as "asc", "desc" is what you're suggesting?

Reply:
The first example is quite common actually.

Reply:
I'm thinking this is not a common use case. The concern is primarily folks thinking `""` are part of the value which would be wrong. Lets get this in, and see if there's a reaction from our consumers. If so, we can use colours to denote string vs integer values.